.github/workflows/validate-merge.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.16.4
+        go-version: 1.19.1
```

Comment:
Please keep internally consistent. If the dockerfil is using 19.2, let' use 19.2 here too.